psi4/driver/procrouting/proc.py

```diff
@@ -1499,21 +1509,27 @@ def scf_helper(name, post_scf=True, **kwargs):
     # The wfn from_file routine adds the npy suffix if needed, but we add it here so that
     # we can use os.path.isfile to query whether the file exists before attempting to read
     read_filename = scf_wfn.get_scratch_filename(180) + '.npy'
+    read_mos = ((core.get_option('SCF', 'GUESS') == 'READ') and os.path.isfile(read_filename)) or ref_wfn is not None
+    origin = "wfn" if ref_wfn is not None else "file"
+    if read_mos:
+        if ref_wfn is not None:
+            old_wfn = ref_wfn
```

Comment:
Yes correct. I tried to avoid duplicating code and do both cases at once. If we keep this option, I still have to check if everything works as it should.